test/unit/dsl/objects_test.rb

```diff
@@ -78,6 +78,40 @@
     end
 
     it 'constructs a simple resource+argument with method calls' do
+      obj.qualifier = [['resource'], ['hello', 'world']]
```

Comment:
Those tests are great, but they are not testing the be_in matcher. Instead they only verify that the inspec obects code generates the right InSpec ruby code. Instead I propose to create a new `test/integration/matcher_test.rb`. This will run RSpec and test the matcher. See https://github.com/chef/inspec/blob/master/test/integration/default/cmp_matcher_spec.rb